common/governments/00_government_types.txt

```diff
@@ -62,21 +62,29 @@ theocracy_government = {
 clan_government = {
 	primary_holding = castle_holding
 	primary_culture_groups = { orc_group dwarf_group tauren_group}
-	
+
+	court_generate_spouses = no
+	court_generate_commanders = yes
+
+	supply_limit_mult_for_others = 0		
+	affected_by_development = yes		
+	regiments_prestige_as_gold = yes
+
 	fallback = 2
-	
-	# Warcraft
-	# preferred_religions = { islam_religion }
 
 	create_cadet_branches = yes
 	rulers_should_have_dynasty = yes
+	council = yes
 
-	valid_holdings = { castle_holding }
-	required_county_holdings = { castle_holding city_holding church_holding }
+	valid_holdings = { castle_holding tribal_holding }
+	required_county_holdings = { castle_holding }
 	always_use_patronym = yes
-	dynasty_named_realms = yes
+	dynasty_named_realms = no
 
-	vassal_contract = { clan_government_obligations }
+	vassal_contract = 
+	{ 
+		clan_government_obligations
+	}
```

Comment:
Wha? It doesn't look like the vanilla clan government at all now.
Regiments cost prestige?